tests/common/helpers/enable-dynamic-buffer.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import subprocess
+import re
+
+from swsscommon.swsscommon import ConfigDBConnector
+
+lossless_profile_name_pattern = 'pg_lossless_([1-9][0-9]*000)_([1-9][0-9]*m)_profile'
+
+# Connect to the CONFIG_DB
+db_kwargs = {}
+config_db = ConfigDBConnector(**db_kwargs)
+config_db.db_connect('CONFIG_DB')
+
+# Don't enable dynamic buffer calculation if it is a MSFT SKU
+metadata = config_db.get_entry('DEVICE_METADATA', 'localhost')
+if 'ACS-MSN' not in metadata['hwsku']:
+    print("Don't enable dynamic buffer calculation for MSFT SKUs")
```

Comment:
please change to non-default SKUs.
default SKUs should have it enabled, non default can keep it disabled.